### DISCUSSION TITLE
Security: Access token example value

### DISCUSSION THREAD

**Discussion by Rajesh-KG** [OAI/OpenAPI-Specification#2813] [Q&A]
I've a requirement to provide example value for access token for Oauth type security schemes.
My request will look like this.

curl
-X **POST**
-H "Content-Type: **application/json**"
-H "Authorization: **my-oauthtoken 0000.xxxxxxxxx**"

I tried to set **Authorization** header in parameters object. But, it violates the openapi specification.
And there is no way to provide 'example' in securityScheme object. Is there any alternative ways to provide example for Authorization.

Notes auth type:
- **Authorization type** : OAuth2 
- **Flow** : Authorization code

Thanks in advance!

**Comment by LasneF** [comment]
hello @Rajesh-KG , 
as your authorization type is Oauth2 with Authorization code ,  this is related to https://www.rfc-editor.org/rfc/rfc6749 this means thatt the token header format is defined by the specification  ; you are able to get the Authorization  header by leveraging the token_type used replied when you get the access token in 99 % of the case it will be bearer as per definition of the https://www.rfc-editor.org/rfc/rfc6750 

and so the authorization header will be in 99% of the case 
-H "Authorization: bearer 0000.xxxxxxxxx"

then as of today there is several way to document specific , you can leverage the description field , and or the bearerFormat to provide custom format if required (this is unlikely) but still Oauth2 mentionned the mac token

**Comment by handrews** [comment]
Assuming this was sufficiently answered due to lack of further response; closing.
